my_list.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'my_enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
```

Comment:
- [ ] MyList class should have the #initialize method that accepts multiple arguments, not just one argument, kindly consider implementing this so your class accepts more than one argument

e.g
```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```